requirements.txt

```diff
@@ -4,4 +4,4 @@ xarray>=0.15
 dask[complete]>=2.6
 netCDF4>=1.4
 python-dateutil>=2.8.1
-cf-xarray>=0.3.1
+cf-xarray>=0.3.1,<=0.8.4
```

Comment:
This has repercussions downstream. As far as I can tell, the major issue is that cf_xarray is breaking in Python3.8. Could we add this upper pin only for a specific Python? I've started to do this for a few projects already.

Reply:
```suggestion
cf-xarray>=0.3.1,<=0.8.4; python_version == '3.8'
cf-xarray>=0.3.1; python_version >= '3.9'
```